mumd/src/audio.rs

```diff
@@ -216,10 +190,55 @@ impl Audio {
 
         output_stream.play().unwrap();
 
-        let sounds = EVENT_SOUNDS
+        let mut res = Self {
+            output_config,
+            _output_stream: output_stream,
+            _input_stream: input_stream,
+            input_volume_sender,
+            input_channel_receiver: Some(input_receiver),
+            client_streams,
+            sounds: HashMap::new(),
+            output_volume_sender,
+            user_volumes,
+            play_sounds,
+        };
+        res.load_sound_effects(&vec![]);
+        res
+    }
+
+    pub fn load_sound_effects(&mut self, sound_effects: &Vec<SoundEffect>) {
```

Comment:
Instead of indicating the type as `&Vec<..>`, consider using `&[..]` since it's both more idiomatic and allows more datatypes to be used when calling this function.